packages/passport/sdk/src/guardian/guardian.ts

```diff
@@ -227,8 +229,9 @@ export default class GuardianClient {
   }
 
   private async evaluateMessage(
-    { chainID, payload, user }:GuardianMessageValidationParams,
+    { chainID, payload }:GuardianMessageValidationParams,
   ): Promise<guardian.MessageEvaluationResponse> {
+    const user = await this.authManager.getUser() as UserZkEvm;
```

Comment:
is the `this.authManager.getUser()` can also returns null?

Reply:
Only if the user is not logged in, and `eth_signTypedData` checks that the user is logged in before we get here, so this call should never return null.

Reply:
Yes, understand that would not return null as `eth_signTypedData` checks existed.
But **validateMessage** is a public method for the **guardian** class, meaning that anyone/functions can call it. We should also check/validate getUser here to avoid potential errors.